src/app/datapoints-graph/datapoints-graph-view/datapoints-graph-widget-view.component.html

```diff
@@ -82,62 +88,174 @@
-        style="padding-top: 3px; padding-bottom: 2px"
-      >
-        <i
-          c8yIcon="circle"
-          class="m-r-4 icon-14"
-          [ngStyle]="{
-            color: datapoint.color,
-          }"
-        ></i>
-        <span
-          class="text-truncate"
-          [ngClass]="{ 'text-muted': !datapoint.__active }"
+  <div class="d-flex" style="align-items: center">
+    <button
+      class="btn-help btn-help--sm m-r-8"
+      [attr.aria-label]="'Help' | translate"
+      [popover]="legendHelpTemplate"
+      [adaptivePosition]="false"
+      placement="bottom"
+      container="body"
+      triggers="focus"
+    ></button>
+    <div class="inner-scroll">
+      <div class="flex-grow p-t-8 d-flex a-i-start gap-8 p-b-4">
+        <div
+          class="c8y-datapoint-pill flex-no-shrink"
+          *ngFor="let datapoint of displayConfig.datapoints"
+          [ngClass]="{ active: datapoint.__active }"
+          title="{{ datapoint.label }} - {{ datapoint.__target.name }}"
         >
-          <span class="text-truncate">
-            {{ datapoint.label }}
-          </span>
-          <small class="text-muted text-10">
-            {{ datapoint.__target.name }}
-          </small>
-        </span>
-        <i
-          *ngIf="datapointsOutOfSync.get(datapoint)"
-          [tooltip]="
-            'Measurements received for this data point may be out of sync.'
-              | translate
-          "
-          container="body"
-          [adaptivePosition]="false"
-          c8yIcon="exclamation-triangle"
-          class="text-warning m-l-4"
-        ></i>
+          <button
+            type="button"
+            title="{{
+              (datapoint.__active ? hideDatapointLabel : showDatapointLabel)
+                | translate
+            }} "
+            class="c8y-datapoint-pill__btn"
+            (click)="toggleChart(datapoint)"
+          >
+            <i
+              [c8yIcon]="
+                datapoint.__active ? 'eye text-primary' : 'eye-slash text-muted'
+              "
+              class="icon-14"
+            ></i>
+          </button>
+          <div class="c8y-datapoint-pill__label c8y-datapoint-pill__btn">
+            <i
+              c8yIcon="circle"
+              class="m-r-4 icon-14"
+              [ngStyle]="{
+                color: datapoint.color,
+              }"
+            ></i>
+            <span
+              class="text-truncate"
+              [ngClass]="{ 'text-muted': !datapoint.__active }"
+            >
+              <span class="text-truncate">
+                {{ datapoint.label }}
+              </span>
+              <small class="text-muted text-10">
+                {{ datapoint.__target.name }}
+              </small>
+            </span>
+            <i
+              *ngIf="datapointsOutOfSync.get(datapoint)"
+              [tooltip]="
+                'Measurements received for this data point may be out of sync.'
+                  | translate
+              "
```

Comment:
```suggestion
              [tooltip]="
                {{'Measurements received for this data point may be out of sync.'
                  | translate}}
              "
```

Reply:
same as in datapoints-graph-widget-view.component.html